nuleaf-source/controllers/events.js

```diff
@@ -20,13 +20,17 @@ var EventsDAO = require('../database/events');
  *   start_date: Filter for events after this date.
  *   end_date  : Filter for events before this date.
  *   location  : Filter for events with this location.
+ *   skip      : The amount of events skipped search on a certain page.
```

Comment:
Although skip and limit is primarily used for pagination, they could be used individually just to limit the number of items returned or skip an arbitrary number of items. We should avoid comments that refer to pagination for this reason.